scipy/stats/tests/test_distributions.py

```diff
@@ -1377,7 +1377,30 @@ def test_sf_isf(self):
         xx = stats.gumbel_l.isf(y)
         assert_allclose(x, xx)
 
+        
+class TestGammaGompertz(object):
+    def test_pdf(self):
+        vals = gamma_gompertz.pdf(np.arange(0., 5., 1.), 1., 1.)
+        expected = [1., 0.367879441, 0.135335283, 0.0497870684, 0.0183156389]
```

Comment:
needs a comment on where these numbers come from (an alternative implementation? paper and pencil?)